filters/filter_24_QuickFixes/metadata.json

```diff
@@ -0,0 +1,18 @@
+{
+  "filterId": 24,
+  "name": "AdGuard Quick Fixes filter",
+  "description": "Filter to quickly resolve content filtering issues on popular websites without updating the extension.",
+  "timeAdded": 1724747105555,
+  "homepage": "https://adguard.com/kb/general/ad-filtering/adguard-filters/",
```

Comment:
kb should be updated

Reply:
https://github.com/AdguardTeam/KnowledgeBase/pull/560